src/dep/triemap/triemap.c

```diff
@@ -223,42 +223,17 @@ void *TrieMapNode_Find(TrieMapNode *n, char *str, tm_len_t len) {
       // let's find a child to continue to
       tm_len_t i = 0;
       TrieMapNode *nextChild = NULL;
-      // if (!(n->flags & TM_NODE_SORTED) && n->numChildren > 2) {
```

Comment:
Let's not touch this file... It might be commented out, but it might also be useful to include it back in?

Reply:
Done